setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(name='hetmech',
+      description='A search engine for hetnets',
+      long_description='Matrix implementations of path-count-based measures',
+      url='https://github.com/greenelab/hetmech',
+      license='BSD 3-Clause License',
+      packages=['hetmech'],
+      install_requires=['hetio==0.2.8', 'numpy==1.13.1', 'scipy==0.19.1',
+                        'xarray==0.9.6']
```

Comment:
Let's put each install on its own line. Also remove versions, unless we know we need a min version for a certain package, and then use >=. Whereas environment.yml is meant to define an environment as specifically as possible, setup.py should define it as loosely as possible for greatest compatibility.

Reply:
Makes sense. I left the hetio version in there because we migrated some functions. Do you think it's necessary or could that version label be removed as well?

Reply:
Yep the min version is the right move for hetio since we use functionality that only exists since 0.2.8.